src/App.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom'
+import './App.css';
+import Books from './components/Books/Books';
+import Categories from './components/Categories/Categories';
+
+function App() {
+  return (
+    <div className="App">
+      <nav>
+        <h1>Bookstore CMS</h1>
+        <ul>
+          <li><Link to="/">Books</Link></li>
+          <li><Link to="/Categories">Categories</Link></li>
+        </ul>
+      </nav>
+      <Routes>
+        <Route path="/" element={<Books />} />
```

Comment:
- [ ] In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page. To make it functional you should use the `Route exact` prop.

eg: 
```jsx
<Route exact path="/">
      <BookList />
</Route>
```